# Generated by Django 5.0.2 on 2024-11-03 15:42

import django.contrib.auth.validators
import django.core.validators
from django.db import migrations, models


class Migration(migrations.Migration):

    dependencies = [
        ("registration", "0009_alter_user_full_name"),
    ]

    operations = [
        migrations.AlterField(
            model_name="user",
            name="username",
            field=models.CharField(
                error_messages={"unique": "もうすでに存在しているユーザー名です。"},
                help_text="必須項目。5文字以上50文字以内で、半角英数字と、@/./+/-/_ の記号のみで入力してください。",
                max_length=50,
                unique=True,
                validators=[
                    django.core.validators.MinLengthValidator(5),
                    django.contrib.auth.validators.UnicodeUsernameValidator(),
                ],
                verbose_name="ユーザーID",
            ),
        ),
    ]
